cmd/server/main.go

```diff
@@ -154,7 +156,12 @@ func main() {
 
 	proxyCache := cacheConfigBuilder()
 
-	adapterConfig := threescale.NewAdapterConfig(proxyCache, parseMetricsConfig())
+	grpcKeepAliveFor := time.Minute
+	if viper.IsSet("grpc_conn_max_minutes") {
```

Comment:
Is this really the maximum amount of minutes a connection will be kept alive for? Perhaps rename to something conveying the fact it is a keep-alive and not a maximum but a setting: connections will try to be kept for these many minutes, no less and (hopefully) no more?

Reply:
Ok. Any suggestions for wording?

Reply:
Not much comes to mind. Maybe `grpc_keepalive_minutes`? Since this is just passing on the computed result to another API, maybe they have wording that can be useful for us?